terraform/modules/self-service/iam.tf

```diff
@@ -190,15 +190,17 @@ data "aws_iam_policy_document" "access_config_metadata" {
     sid       = "AllowGetAndPutObject"
     effect    = "Allow"
     resources = [
+      "${aws_s3_bucket.config_metadata.arn}",
       "${aws_s3_bucket.config_metadata.arn}/*",
-      "${data.aws_ssm_parameter.integration_metadata_bucket.value}/*"
+      "arn:aws:s3:::govukverify-${local.service}-integration-config-metadata",
```

Comment:
this will give access to the bucket from all the environments... should we pass it in per environment somehow?